src/main/webapp/vendor/startbootstrap/template.jsp

```diff
@@ -191,9 +186,14 @@
     <script src="https://uicdn.toast.com/tui-calendar/latest/tui-calendar.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/luxon@1.24.1/build/global/luxon.min.js"></script>
 
+    <!-- Google OAuth Script -->
+    <script src="https://apis.google.com/js/platform.js" async defer></script>
+
     <!-- Our scripts -->
     <script type="module" src="js/courseSelector.js"></script>
     <script type="module" src="js/${param.scriptFile}.js"></script>
+    <script src="js/pages/login.js"></script>
```

Comment:
can this be a module too? otherwise you run into the same namespacing issues we discussed a few weeks back.

Also, regarding the "is this a page" discussion, I think this this isn't actually a page.